wordpress-mobile/WordPress-iOS
# Comment Detail: Backport arrowshape.turn.up.backward for iOS 13

Refs #17087

As titled, the `arrowshape.turn.up.backward` symbol is backported for iOS 13 into our assets catalog since we are still targeting iOS 13. Here's a screenshot:

Before | After | iOS 14 (for reference)
--|--|--
![before_ios13](https://user-images.githubusercontent.com/1299411/141515014-bbe83660-b1c7-43b4-9e20-49f592730231.png) | ![after_ios13](https://user-images.githubusercontent.com/1299411/141515022-6afdd809-3b2c-4106-a28c-898ccdc17499.png) | ![ios14](https://user-images.githubusercontent.com/1299411/141515186-c357d714-66e6-4105-8bcf-cabfac969040.png)

---

⚠️ NOTE: Auto-merge is enabled.

---

## To test

- Enable `newCommentDetail` feature flag.
- In iOS 13, Go to My Site > Comments, and select any comment.
- Ensure that the Reply icon is displayed.

## Regression Notes
1. Potential unintended areas of impact
n/a. Feature is unreleased.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
n/a. Feature is unreleased.

3. What automated tests I added (or what prevented me from doing so)
n/a. Feature is unreleased.

PR submission checklist:
